tests/testthat/helper-tune-package.R

```diff
@@ -26,3 +26,46 @@ helper_objects_tune <- function() {
     svm_mod = svm_mod
   )
 }
+
+# When not interactive, the cli progress bar is occasionally "reset" at random
+# intervals throughout tuning, which means that intermediate counts may appear
+# in snapshots. In an output like:
+#
+# > A | error:   AHHhH
+# There were issues with some computations   A: x2
+# There were issues with some computations   A: x4
+# There were issues with some computations   A: x5
+# There were issues with some computations   A: x5
+#
+# ...we just want to see the unique issues, e.g.
+#
+# > A | error:   AHHhH
+catalog_lines <- function(lines) {
+  lines[grepl("^>", lines)]
+}
```

Comment:
Note that since this function is in a `tests/testthat/helper-` file, it will be available on `devtools::load_all()`. It also is not available in extratests, as it would be if it was in `R/`, so we have to copy the function in the test file in the linked PR.

I've kept this one-liner as a function instead of inlining it so that we can Cmd + Click it in tests and see the context above.